cfme/control/explorer/alert_profiles.py

```diff
@@ -147,6 +171,47 @@ def delete(self, cancel=False):
             view.flash.assert_message(
                 'Alert Profile "{}": Delete successful'.format(self.description))
 
+    @property
+    def exists(self):
+        """Check existence of this Alert Profile.
+
+        Returns: :py:class:`bool` signalizing the presence of the Alert Profile in database.
+        """
+        miq_sets = self.appliance.db["miq_sets"]
+        return self.appliance.db.session\
+            .query(miq_sets.description)\
+            .filter(
+                miq_sets.description == self.description and miq_sets.set_type == "MiqAlertSet")\
+            .count() > 0
+
+    def assign_to(self, assign, selections=None, tag_category=None):
+        """Assigns this Alert Profile to specified objects.
+
+        Args:
+            assign: Where to assign (The Enterprise, ...).
+            selections: What items to check in the tree. N/A for The Enteprise.
+            tag_category: Only for choices starting with Tagged. N/A for The Enterprise.
+        """
+        view = navigate_to(self, "Edit assignments")
+        changed = view.fill({
+            "assign_to": assign,
+            "tag_category": tag_category,
+            "selections": selections
+        })
```

Comment:
Since hte pattern below will be, I assume, quite common, @mfalesni does it make sense to have an action on fill, like before, and that if nothing changed it won't invoke it, but if something did, then it will?